packages/apps/graph/generated-schema.graphql

```diff
@@ -168,21 +191,50 @@ type PageInfo {
 scalar PositiveFloat
 
 type Query {
-  account(accountName: String!, moduleName: String!): ModuleAccount!
+  """Find an account by its name and its module, such as coin."""
+  account(accountName: String!, moduleName: String!): ModuleAccount
+
+  """Find a block by its hash."""
```

Comment:
```suggestion
  """Retrieve a block by hash."""
```
I dont know wheter all `find`s can be `retrieve`s. Find sounds more "voluntary". Bikeshedding though :)